data-raw/dataprep.R

```diff
@@ -11,8 +11,18 @@ squirrels <- read_csv(here::here("data-raw", "2018_Central_Park_Squirrel_Census_
   rename(
     long = x,
     lat = y
+  ) %>%
+  mutate(date = as.character(date)) %>%
+    mutate(
```

Comment:
Can you fix the indentation here?

Reply:
Should be looking better now @mine-cetinkaya-rundel